ui/admin/app/controllers/scopes/scope/aliases/index.js

```diff
@@ -26,6 +27,28 @@ export default class ScopesScopeAliasesIndexController extends Controller {
   @tracked page = 1;
   @tracked pageSize = 10;
 
+  /**
+   * If can list (at least): return default welcome message.
+   * If can create (only): return create-but-not-list welcome message.
+   * If can neither list nor create: return neither-list-nor-create welcome message
+   * @type {string}
+   */
+  get messageDescription() {
```

Comment:
None blcoking comment: 
I am totally fine with this implementation, IMHO its a big DX win over what we had before and I am gratefull to see this amazing work happening.

What is you opinion on something like this? Your work provided a huge improvement on readability and IMHO a good separation of concerns. But I though about asking your opinion on this change. 

But please, as I said, I am totally fine with your implementation 😉 

```javascript
get messageDescription() {
    const canList = this.can.can('list model', this.scope, { collection: 'aliases' });
    const canCreate = this.can.can('create model', this.scope, { collection: 'aliases' });
    const resource = this.intl.t('resources.alias.title_plural');
    let description = 'descriptions.neither-list-nor-create';

    if (canList) {
      description = 'resources.alias.messages.none.description';
    } else if (canCreate) {
      description = 'descriptions.create-but-not-list';
    }

    return this.intl.t(description, { resource });
  }
```

Reply:
Thank you for the suggestion & example! I think that ur recommendation makes it easier to read! I'll implement this, should be pretty quick change 👍

Reply:
updated!